src/OrchardCore/OrchardCore.Abstractions/Extensions/DocumentJsonSerializerOptionsConfiguration.cs

```diff
@@ -22,6 +22,7 @@ public void Configure(DocumentJsonSerializerOptions options)
         options.SerializerOptions.PropertyNameCaseInsensitive = JOptions.Base.PropertyNameCaseInsensitive;
         options.SerializerOptions.AllowTrailingCommas = JOptions.Base.AllowTrailingCommas;
         options.SerializerOptions.WriteIndented = JOptions.Base.WriteIndented;
+        options.SerializerOptions.PreferredObjectCreationHandling = JsonObjectCreationHandling.Populate;
```

Comment:
Why not this?
```suggestion
        options.SerializerOptions.PreferredObjectCreationHandling = JOptions.Base.PreferredObjectCreationHandling;
```
As I mentioned in the other PR, this setting is already applied to Base.

Reply:
Didn't realize the other lines were copying the base options.